docs/sdk/server-side-sdks/node.md

```diff
@@ -113,11 +125,36 @@ See [getVariables](https://docs.devcycle.com/bucketing-api/#operation/getVariabl
 
 You can fetch all segmented features for a user:
 
+[DVCFeature Typescript Schema](https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/nodejs/src/types.ts#L193)
+
 ```javascript
 const features = dvcClient.allFeatures(user)
 ```
 See [getFeatures](https://docs.devcycle.com/bucketing-api/#operation/getFeatures) on the Bucketing API for the feature response format.
 
+### Tracking User Events
+
+Track a custom event for a user, pass in the user and event object.
+
+Calling Track will queue the event, which will be sent in batches to the DevCycle servers.
+
+[DVCEvent Typescript Schema](https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/nodejs/src/types.ts#L166)
+
+```typescript
+const event: DVCEvent = {
+    type: 'customType',
+    target: 'new_subscription',
+    value: 100.1,
+    date: Date.now()
+}
+dvcClient.track(user, event)
+```
+
+### Flush Events
+
+If you would like to force a flush of events in the event queue, you can call `flushEvents()`. 
```

Comment:
This section should mention that an error will result if flushEvents() is called when there is a flush in progress.

Also, shouldn't we expose a function to check if a flush is in progress, if we expose the ability to manually flush?

Reply:
the user-facing flush method should not cause an error to happen if a flush is in progress.
The only thing it should do is make sure that if the current flush in progress finishes, and events are in the queue, another flush will happen immediately.

Reply:
Currently, it just logs an error if it receives an error from the WASM: 
<img width="794" alt="image" src="https://user-images.githubusercontent.com/1219069/191979513-27ef00d7-e479-4d33-8734-67123be59487.png">